configure.ac

```diff
@@ -639,6 +644,7 @@ AC_CHECK_HEADERS([arpa/inet.h fcntl.h limits.h malloc.h memory.h netdb.h \
                   sys/socket.h sys/time.h unistd.h])
 
 AC_CHECK_HEADERS([pthread.h], [], [AC_MSG_ERROR([pthread.h cannot be found!])])
+AC_CHECK_HEADERS([isa-l.h],   [], [AC_MSG_ERROR([isa-l.h cannot be found!])])
```

Comment:
As per our discussion, we should check if intel isa-l exists.
* Yes, then use the Intel ISA-L APIs.
* No, then use Galois APIs.

I think we need to revise this file a bit.

Reply:
Yes. Working on it.

Reply:
For example, If Intel ISA is available in the development system, we will define:
HAVE_ISAL